packages/ui/src/pages/Profile/Accounts.tsx

```diff
@@ -17,14 +19,14 @@ export function Accounts({ accounts }: Props) {
         </tr>
       </thead>
       <tbody>
-        {accounts.map((account) => (
+        {allAccounts.map((account) => (
           <BodyRow key={account.address}>
             <BodyCell>
               <h3>{account.name}</h3>
               <p>{account.address}</p>
             </BodyCell>
             <BodyCell>
-              <p>0 JOY</p>
+              <p>{balances.map[account.address] ? balances.map[account.address].total : '-'}</p>
```

Comment:
This seems to complex here. On a language level I guess we can do sth like
```suggestion
              <p>{balances.map[account.address].total || '-'}</p>
```
But perhaps our hook can return a more convenient object that would handle rendering? Or we could have a small component for balance that would have the view logic.

Reply:
For sure more atomic components are welcome. However as this is still work in progress I could introduce it later when implementing the rest of balances. I'm also thinking about introducing a component for the whole account as it will have some "complex" logic.

> This seems to complex here. On a language level I guess we can do sth like

👌 I still forgot that I can use this syntax in TS